<launch>
  <arg name="debug" default="false"/>
  <arg if="$(arg debug)" name="launch_prefix" value="xterm -rv -e gdb -ex run -args"/>
  <arg unless="$(arg debug)" name="launch_prefix" value=""/>

<!--
  <node node="AR_Kinect_multi" pkg="ar_pose" type="ar_multi" respawn="false" output="screen">
    <rosparam command="load" file="$(find ar_object_tracker)/config/ar_single_marker.yaml"/>
    <param name="camera_frame" type="string" value="/openni_rgb_optical_frame"/>
    <param name="marker_frame" type="string" value="ARMarker"/>
    <param name="camera_image_topic" type="string" value="/camera/rgb/image_color"/>
    <param name="camera_info_topic" type="string" value="/camera/rgb/camera_info"/>
    <param name="publish_tf" type="bool" value="true"/>
  </node>
  -->

  <node name="world_to_base" pkg="tf" type="static_transform_publisher"
        args="0 0 0 0 0 0 world base_link 100" />

  <node name="ar_object_tracker" pkg="ar_object_tracker" type="object_tracker">
    <param name="test_mode" type="bool" value="true"/>
  </node>
</launch>
